README.md

```diff
@@ -29,7 +29,7 @@ The only required option is `orgId`, all others are optional.
 *  `orgId` -  Sets your FullStory Org Id. Find out how to get your Org Id [here](https://help.fullstory.com/hc/en-us/articles/360047075853).
 *  `debug` - When set to `true`, enables FullStory debug messages; defaults to `false`.
 * `namespace` - Sets the global identifier for FullStory when conflicts with `FS` arise; see [help](https://help.fullstory.com/hc/en-us/articles/360020624694-What-if-the-identifier-FS-is-used-by-another-script-on-my-site-).
-* `recordCrossDomainIFrames` - When set to `true`, FullStory is added to cross-domain IFrames and records content; defaults to `false`. Before using, you should understand the security implications, and configure your [Content Security Policy](https://www.html5rocks.com/en/tutorials/security/content-security-policy/) (CSP) HTTP headers accordingly - specifically the frame-ancestors directive. Failure to configure your CSP headers while using this setting can bypass IFrames security protections that are included in modern browsers.
+* `recordCrossDomainIFrames` - FullStory can record cross-domain iFrames if: 1. The FullStory Browser SDK is running in the cross-domain iFrame and 2. `recordCrossDomainIFrames` is set to `true` in the cross-domain iFrame and 3. The FullStory Browser SDK is running in the parent page of the cross-domain iFrame. Click [here](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) for a detailed explanation of what "cross-domain" means. Before using, you should understand the security implications, and configure your [Content Security Policy](https://www.html5rocks.com/en/tutorials/security/content-security-policy/) (CSP) HTTP headers accordingly - specifically the frame-ancestors directive. Failure to configure your CSP headers while using this setting can bypass IFrames security protections that are included in modern browsers. Defaults to `false`. 
```

Comment:
This description is much clearer; nice.

It might be good to include a link to [our KB article on this topic](https://help.fullstory.com/hc/en-us/articles/360020622514-Can-FullStory-capture-content-that-is-presented-in-iframes-#2-the-outer-page-is-running-fullstory-and-you-have-iframes-runni), since it provides some additional details. We could even simplify the explanation here some and just advise folks that enabling this option is a potential security issue, and that they should consult the KB article before turning it on. (I do find it worrisome that both this paragraph and the KB article say "you should understand the security implications", but neither actually explains what they are. The KB advises to contact us first if anything is unclear, which is a good idea.)

Reply:
I can include a link to the KB article. One thing I'll want to clarify is that that `recordCrossDomainIFrames` abstracts `_fs_run_in_iframe` so to be aware when reading the KB. Let me see what I can do.

Reply:
@sethfowler - please check out the latest langauge.

Reply:
LGTM!